### PR TITLE
Cleanup init

### PR DESCRIPTION
**Pull request by bfred-it**
This is mostly moving code around. If you read this code, do [commit by commit, possibly without whitespace.](https://github.com/sindresorhus/refined-github/pull/784/commits/063a9ab2a11bd00f50be35f5436aec6e737d1900?w=1)

### CODE REVIEW

File: src/content.js [hunk 1]
```diff
@@ -71,6 +72,20 @@ function hideEmptyMeta() {
...
+
+	if (hideStarsOwnRepos) {
+		const username = getUsername();
+		observeEl('#dashboard .news', () => {
+			$('#dashboard .news .watch_started, #dashboard .news .fork')
+				.has(`a[href^="/${username}"]`)
+				.css('display', 'none');
+		});
+	}
+}
```
Comment (sindresorhus) [inline src/content.js]: Move it to its own file? Would be nice to eventually reduce `content.js` to just be a router/delegator.

Comment (bfred-it) [reply src/content.js]: That was the idea on my first refactor, but I'm afraid that we'd have too many 5-line files on the `libs` folder. Surely a lot of the code we have now in `content` is long enough to be in their own file.

Comment (sindresorhus) [reply src/content.js]: > but I'm afraid that we'd have too many 5-line files on the libs folder

I don't see that as a problem.

---

I also think we should create a separate directory for the features. Now they're mixed with utils, icons, domify, etc.

Comment (bfred-it) [reply src/content.js]: I might do that after we close a few PRs

**Pull request merged**